docs/public-networks/reference/cli/options.md

```diff
@@ -3074,6 +3074,46 @@ Enabling revert reason may use a significant amount of memory. We don't recommen
 
 :::
 
+### `rpc-gas-cap`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--rpc-gas-cap=<INTEGER>
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--rpc-gas-cap=50000000
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+BESU_RPC_GAS_CAP=50000000
+```
+
+</TabItem>
+
+<TabItem value="Configuration file" label="Configuration file">
+
+```bash
+rpc-gas-cap=50000000
+```
+
+</TabItem>
+
+</Tabs>
+
+Sets a limit on the amount of gas that can be used for executing a transaction using HTTP JSON-RPC methods. This limit prevents gas amounts from being set too high, which can cause network congestion and increase transaction costs.
```

Comment:
```suggestion
Sets a cap for transaction simulation RPC methods. Must be >=0. The default is 0 which means no limit. This cap prevents excessive resource usage from eth_call requests.
```

Reply:
you could link to eth_call

Reply:
this cap is only for transaction simulation, not actual transaction execution

Reply:
Thanks so much! Super helpful

Reply:
Is this new wording accurate.. or should I add exactly what you have?

Reply:
looks good